hack/dashboard/openshift/deploy-grafana.sh

```diff
@@ -277,12 +277,18 @@ setup_grafana_dashboard() {
 	ok "created grafana datasource \n"
 
 	info "Creating dashboard config map"
-	oc_delete_grafana_ns configmap grafana-dashboard-cm
-	oc_create_grafana_ns configmap grafana-dashboard-cm --from-file=hack/dashboard/assets/dashboard.json
+	oc_delete_grafana_ns configmap kepler-dashboard-cm
+	oc_create_grafana_ns configmap kepler-dashboard-cm --from-file=hack/dashboard/assets/kepler/dashboard.json
+
+	oc_delete_grafana_ns configmap prometheus-dashboard-cm
+	oc_create_grafana_ns configmap prometheus-dashboard-cm --from-file=hack/dashboard/assets/prometheus/dashboard.json
```

Comment:
I think openshift already ships something similar to this under `Observe/Dashboards`. Could you please confirm this provides more insight?

Reply:
yes this has more panels than openshift one, the above pics are not complete panels. since this is grafana, users can also add their own panels.

Reply:
Lets add them ...